client/datasync-client/integration/integration.ts

```diff
@@ -0,0 +1,53 @@
+import * as sync from 'fh-sync-js';
+
+// Provide backwards compatibility with documentation and examples
+const $fh = { sync };
+const datasetId = 'UserTasks';
+
+const options: sync.SyncOptions = {
+  cloudUrl: 'http://localhost:3000',
+  sync_frequency: 10,
+  storage_strategy: 'dom'
+};
+
+$fh.sync.init(options);
+
+const queryParams = {};
+const metaData = {};
+
+const task: any = {
+  name: 'test task',
+  status: 'finished'
+};
+
+$fh.sync.manage(datasetId, options, queryParams, metaData, function() {
+  $fh.sync.doCreate(datasetId, task, function(data) {
+    // tslint:disable-next-line:no-console
```

Comment:
This is special comment that disables tslint rule: 
https://palantir.github.io/tslint/usage/rule-flags/

That's integration file so we actually want to use console.log